src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,165 @@
+package main.java.duke;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Parser {
+
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+
+    public Parser() {
+
+    }
+
+    public static ArrayList<Task> getTaskList() {
+        return taskList;
+    }
+
+    public static void addTask(Task newTask) {
+        taskList.add(newTask);
+        System.out.println("Got it. I've added this task:" + "\n" + newTask.toString() +
+                "\n" + "Now you have " + taskList.size() + " tasks in the list.");
+    }
+
+    public static void listTask() {
+        System.out.println("Here are the task in your list:");
+        for(int i = 0; i < taskList.size(); ++i) {
+            System.out.println(i+1 + ". " + taskList.get(i));
+        }
+    }
+
+    public static void deleteTask(int i) {
+        Task task = taskList.get(i-1);
+        taskList.remove(i-1);
+        System.out.println("Noted. I've removed this task:" + "\n" + task.toString() +
```

Comment:
Should the `+` operator be on the next line?
 I have noted the issue in several other places as well...